modules/EmbedPlayer/resources/mw.EmbedPlayerKplayer.js

```diff
@@ -146,7 +146,6 @@
 						'switchingChangeStarted': 'onSwitchingChangeStarted',
 						'switchingChangeComplete': 'onSwitchingChangeComplete',
 						'flavorsListChanged': 'onFlavorsListChanged',
-						'enableGui': 'onEnableGui',
 						'liveStreamOffline': 'onLiveEntryOffline',
```

Comment:
@itaykinnrot - as suspected, this breaks UI disabling during ads when using KPlayer. We need to look into it and find why the JS disable doesn't kick-in before we can merge this.